opennms-model/src/main/java/org/opennms/netmgt/model/events/snmp/SyntaxToEvent.java

```diff
@@ -111,7 +111,13 @@ public static Parm processSyntax(final String name, final SnmpValue value) {
             if (m_syntaxToEvents[i].getTypeId() == -1 || m_syntaxToEvents[i].getTypeId() == value.getType()) {
                 val.setType(m_syntaxToEvents[i].getType());
                 String encoding = null;
-                if (value.isDisplayable()) {
+                boolean displayable = false;
+                try {
+                    displayable = value.isDisplayable();
+                } catch (ArrayIndexOutOfBoundsException aioobe) {
+                    // Eat it
```

Comment:
Can you put a comment here that says "This should not be necessary when NMS-7547 is fixed"?